GOVERNANCE.md

```diff
@@ -3,43 +3,58 @@ the Benevolent dictator governance model by Ross
 
-## [Core Developers](https://github.com/orgs/tidyverse/teams/ggplot2)
-Core developers are contributors who have made several valuable contributions to the project and are now relied upon to write code directly to the repository and screen the contributions of others. Core developers are empowered to merge pull requests after careful review. Core developers have no authority over the overall direction of the project, however it is their job to help develop or elicit appropriate contributions to the project. As a matter of policy, core developers, even if no longer active, are listed (chronologically) in [package authors](https://ggplot2.tidyverse.org/authors.html).
+## Users
+
+Anyone who creates visualisations with ggplot2 is a user. They are the most important members of the community: without them, the project would have no purpose. 
 
-An invitation to join the core team will be extended to contributors who have made a major contributions as a contributor. For example, they have added a major new feature or have a consistent pattern of smaller contributions over an extended period of time. Any existing core developer can propose a contributor be invited to the core team, pending confirmation by existing authors.
+Users are encouraged to participate in the life of the project and the community as much as possible. User contributions enable the project team to ensure that they are satisfying the needs of those users. Common user activities include (but are not limited to):
+
+- evangelising about the project
+- asking and answering on community forums
+- providing moral support (a 'thank you' goes a long way)
+
+Users who continue to engage with the project and its community will often find themselves becoming more and more involved. Such users may then go on to become contributors, as described above.
 
 ## Contributors
-Contributors are community members who make valuable contributions, such as those outlined in the list below, but generally do not have the authority to make direct changes to the project code. Instead, contributors can suggest changes to project code through the pull request process outlined in the project's [CONTRIBUTING](https://github.com/tidyverse/ggplot2/blob/master/CONTRIBUTING.md) document. 
 
-Anyone can become a contributor. There is no expectation of commitment to the project, no specific skill requirements and no selection process. To become a contributor, a community member simply has to perform one or more actions that are beneficial to the project.
+Contributors interact with the project on GitHub by filing new issues, improving existing issues, or submitting pull requests. Anyone can become a contributor: there is no expectation of commitment to the project, no specific skill requirements, and no selection process. The only obligation is to follow the [code of conduct](CODE_OF_CONDUCT.md).
```

Comment:
replace "no specific skill requirements" with "no required set of skills" (so that all three clauses are singular to match the opening "there is")